### ISSUE TITLE
fs.move across windows drives fails

### ISSUE THREAD

**Issue by nicks** [jprichardson/node-fs-extra#109]
```js
var x = 'C:\\test.txt'
var y = 'D:\\test.txt'
require('fs-extra').move(x, y, function (err) {
  throw err
})
```

emits the following error:
```
Can't move source into dest if dest is in source.
```

I believe the detection here is incorrect, and should probably use some other mechanism

**Comment by jprichardson** [comment]
hah, ok, I'm on it.

**Comment by jprichardson** [comment]
Published 0.16.3.
